SUMMARY.md

```diff
@@ -22,10 +22,10 @@
   * [Digital Ocean](deployment/digital-ocean/README.md)
     * [Kubernetes Dashboard](deployment/digital-ocean/dashboard/README.md)
     * [HTTPS](deployment/digital-ocean/https/README.md)
-  * [Human Connection](deployment/human-connection/README.md)
-    * [Error Reporting](deployment/human-connection/error-reporting/README.md)
-    * [Mailserver](deployment/human-connection/mailserver/README.md)
-    * [Maintenance](deployment/human-connection/maintenance/README.md)
+  * [Human Connection](deployment/ocelot-social/README.md)
```

Comment:
```suggestion
  * [Ocelot Social](deployment/ocelot-social/README.md)
```

Reply:
i suggest to merge this, but not close #4043 yet

Reply:
This is an issue @Mogge works on separately …
Therefore I thought not to cross his lines … @ulfgebhardt

Reply:
Oh, this is the SUMMARY.md. Thought it would be the main `README.md`.
I have fixed it locally. @ulfgebhardt